packages/shared/src/components/feeds/FeedNav.tsx

```diff
@@ -78,21 +86,43 @@ function FeedNav(): ReactElement {
         }, {})
       : [];
 
-    return {
+    const urls = {
       ...(hasCustomFeedsEnabled
         ? {
             [`${webappUrl}feeds/new`]: FeedNavTab.NewFeed,
           }
         : undefined),
       [`${webappUrl}`]: FeedNavTab.ForYou,
       ...customFeeds,
-      [`${webappUrl}popular`]: FeedNavTab.Popular,
-      [`${webappUrl}upvoted`]: FeedNavTab.MostUpvoted,
-      [`${webappUrl}discussed`]: FeedNavTab.Discussions,
+    };
+
+    if (seoSidebar === SeoSidebarExperiment.V1) {
+      urls[`${webappUrl}${SharedFeedPage.Explore}`] = FeedNavTab.Explore;
+    } else {
+      urls[`${webappUrl}${SharedFeedPage.Popular}`] = FeedNavTab.Popular;
+      urls[`${webappUrl}${SharedFeedPage.Upvoted}`] = FeedNavTab.MostUpvoted;
+    }
+
+    urls[`${webappUrl}${SharedFeedPage.Discussed}`] = FeedNavTab.Discussions;
+
+    if (seoSidebar === SeoSidebarExperiment.V1) {
+      urls[`${webappUrl}tags`] = FeedNavTab.Tags;
+      urls[`${webappUrl}sources`] = FeedNavTab.Sources;
+      urls[`${webappUrl}leaderboard`] = FeedNavTab.Leaderboard;
```

Comment:
```suggestion
      urls[`${webappUrl}users`] = FeedNavTab.Leaderboard;
```

Endpoint is `/users` for the leaderboard